terminal/bundles/org.eclipse.terminal.view.core/META-INF/MANIFEST.MF

```diff
@@ -12,5 +12,10 @@ Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
 Export-Package: org.eclipse.terminal.view.core;version="1.0.0",
  org.eclipse.terminal.view.core.internal;x-internal:=true,
- org.eclipse.terminal.view.core.utils;version="1.0.0"
+ org.eclipse.terminal.view.core.utils;version="1.0.0";
```

Comment:
A version on an internal package is not really useful (or you need to import it with provider version range) and I'm also not a fan of "friends" concept. Either others need to access the API, then it should be a proper API available for everyone or it should not accessed by others (except tests for example).

Also note that for example CDT still has two connector implementations as well, one would then check if they need some classes here as well.